src/mongo/db/storage/tokuft/tokuft_engine_options.cpp

```diff
@@ -95,69 +95,69 @@ namespace mongo {
+        if (params.count("storage.PerconaFT.engineOptions.fsRedzone")) {
+            fsRedzone = params["storage.PerconaFT.engineOptions.fsRedzone"].as<int>();
             if (fsRedzone < 0 || fsRedzone > 100) {
                 StringBuilder sb;
-                sb << "storage.tokuft.engineOptions.fsRedzone must be between 0 and 100, but attempted to set to: "
+                sb << "storage.PerconaFT.engineOptions.fsRedzone must be between 0 and 100, but attempted to set to: "
                    << fsRedzone;
                 return Status(ErrorCodes::BadValue, sb.str());
             }
         }
-        if (params.count("storage.tokuft.engineOptions.journalCommitInterval")) {
-            journalCommitInterval = params["storage.tokuft.engineOptions.journalCommitInterval"].as<int>();
+        if (params.count("storage.PerconaFT.engineOptions.journalCommitInterval")) {
+            journalCommitInterval = params["storage.PerconaFT.engineOptions.journalCommitInterval"].as<int>();
             if (journalCommitInterval < 1 || journalCommitInterval > 300) {
                 StringBuilder sb;
-                sb << "storage.tokuft.engineOptions.journalCommitInterval must be between 1 and 300, but attempted to set to: "
+                sb << "storage.PerconaFT.engineOptions.journalCommitInterval must be between 1 and 300, but attempted to set to: "
                    << journalCommitInterval;
                 return Status(ErrorCodes::BadValue, sb.str());
             }
         }
-        if (params.count("storage.tokuft.engineOptions.lockTimeout")) {
-            lockTimeout = params["storage.tokuft.engineOptions.lockTimeout"].as<int>();
+        if (params.count("storage.PerconaFT.engineOptions.lockTimeout")) {
+            lockTimeout = params["storage.PerconaFT.engineOptions.lockTimeout"].as<int>();
             if (lockTimeout < 0 || lockTimeout > 60000) {
                 StringBuilder sb;
-                sb << "storage.tokuft.engineOptions.lockTimeout must be between 0 and 60000, but attempted to set to: "
+                sb << "storage.PerconaFT.engineOptions.lockTimeout must be between 0 and 60000, but attempted to set to: "
                    << lockTimeout;
                 return Status(ErrorCodes::BadValue, sb.str());
             }
         }
-        if (params.count("storage.tokuft.engineOptions.locktreeMaxMemory")) {
-            locktreeMaxMemory = params["storage.tokuft.engineOptions.locktreeMaxMemory"].as<unsigned long long>();
+        if (params.count("storage.PerconaFT.engineOptions.locktreeMaxMemory")) {
+            locktreeMaxMemory = params["storage.PerconaFT.engineOptions.locktreeMaxMemory"].as<unsigned long long>();
             if (lockTimeout < (100LL<<20)) {
                 warning() << "PerconaFT: locktreeMaxMemory is under 100MB, this is not recommended for production." << std::endl;
             }
         }
         // TODO: MSE-39
```

Comment:
We should probably port this MSE ticket to the new system, just for tracking, then update this TODO comment.